examples/example_push.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#[macro_use]
+extern crate prometheus;
+#[macro_use]
+extern crate lazy_static;
+
+use std::thread;
+use std::time;
+
+use prometheus::{Histogram, Counter};
+
+lazy_static! {
+    static ref PUSH_COUNTER: Counter = register_counter!(
+        opts!(
+            "example_push_total",
+            "Total number of prometheus client pushed."
+        )
+    ).unwrap();
+
+    static ref PUSH_REQ_HISTOGRAM: Histogram = register_histogram!(
+        histogram_opts!(
+            "example_push_request_duration_seconds",
+            "The push request latencies in seconds."
+        )
+    ).unwrap();
+}
+
+fn main() {
+    println!("Pushing, please wait 10 seconds ...");
```

Comment:
> we must tell user to start prometheus server first.

It should be [pushgateway](https://github.com/prometheus/pushgateway), not prometheues server.

> Add a flag for the prometheus server, not fixed 127.0.0.1:9091

`127.0.0.1:9091` is the default address of pushgateway

-------------

I will address comments later.